code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -202,6 +202,25 @@
 	max_heat_protection_temperature = FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/engine/atmos
 
+	//Construction hardsuit
+/obj/item/clothing/head/helmet/space/hardsuit/engine/construct
+	name = "construction hardsuit helmet"
+	desc = "A special helmet designed for work in a hazardous, low-pressure environment. Much of the protective qualities have been given up for superior movement."
+	icon_state = "hardsuit0-construction"
+	item_state = "con_helm"
+	armor = list(melee = 5, bullet = 5, laser = 5, energy = 5, bomb = 5, bio = 100, rad = 0)
```

Comment:
wrap all of these var names in quotes. not doing so will result in the armor not applying at all